src/pages/Landing/index.tsx

```diff
@@ -147,6 +171,12 @@ export default () => {
                 <ColoredTd color={color} dataLabel='isCool'>
                   {isCool ? 'Yup' : 'Totally Not!'}
                 </ColoredTd>
+                <Td key={`kebab-menu-${key}`}>
+
+                  <div>
+                    <button onClick={() => handleDelete(name)}>Delete</button>
+                  </div>
```

Comment:
This works great now!  But we want it to be a kaba.  you can see the kabab here: 

https://v4-archive.patternfly.org/v4/components/table#composable-actions

and if you click the TS button you can see the source code:

![image](https://github.com/content-services/React-typescript-demo/assets/395077/4936222a-4e5a-4cda-9b80-1828bfd0d400)